builtin/logical/pki/path_roles.go

```diff
@@ -460,6 +494,7 @@ type roleEntry struct {
 	KeyUsage              string `json:"key_usage" structs:"key_usage" mapstructure:"key_usage"`
 	OU                    string `json:"ou" structs:"ou" mapstructure:"ou"`
 	Organization          string `json:"organization" structs:"organization" mapstructure:"organization"`
+	GenerateLease         *bool  `json:"generate_lease" structs:"generate_lease"`
```

Comment:
I think the pointer method does the job without ambiguity anywhere. In upgrade case, how do we distinguish `skip_lease_generation` explicitly being set to `false` vs a role entry that has not been upgraded?